service/upload.go

```diff
@@ -59,7 +59,7 @@ func (u *UploadService) ReadFinesExcel(ctx context.Context, r *bytes.Reader) err
 		// Go through all rows
 		for _, row := range rows {
 			// Skip the first row with designation info
-			if row[IDCol] == "ID" || row[IDCol] == "Id" || row[IDCol] == "id" || row[IDCol] == "iD" {
+			if row[FineNumCol] == "ID" || row[FineNumCol] == "Id" || row[FineNumCol] == "id" || row[FineNumCol] == "iD" {
```

Comment:
You can use strings.ToLower(row[FaNCol]) == "id" instead of use those 4 if conditions